src/index.md

```diff
@@ -58,6 +59,86 @@ const end = view(Inputs.date({label: "End", value: getDateXDaysAgo(1) }));
   }</div>
 </div>
 
+<div class="divider"></div>
+
+<h4>Spark Retrieval Result Codes</h4>
+<body>This section shows the Spark Retrieval Result Codes breakdown. You can adjust the date range. Records exist for the last 90 days.</body>
+
+```js
+const combine = (obj, target, keys) => {
+  obj[target] = keys.reduce((acc, key) => acc + (obj[key] || 0), 0)
+  for (const key of keys) {
+    delete obj[key]
+  }
+}
+const clone = obj => JSON.parse(JSON.stringify(obj))
+const tidy = clone(SparkRetrievalResultCodes).flatMap(({ day, rates }) => {
+  for (const [key, value] of Object.entries(rates)) {
+    rates[key] = Number(value)
+  }
+
+  combine(rates, 'HTTP_5xx', [
+    'LASSIE_502',
```

Comment:
Please keep the distinction between `HTTP_` and `LASSIE_` prefixed errors.

Since we use Lassie for Graphsync retrievals, and there isn't much granularity in the status codes, I propose the following mapping:

- LASSIE_504 -> GRAPHSYNC_TIMEOUT
- All other codes prefixed by LASSIE_ -> GRAPSYNC_ERROR

BTW we can abbreviate `ERROR` to `ERR` if it helps with the visualisation.